annotator-models/trainer/dawid_skene.py

```diff
@@ -204,15 +209,16 @@ def m_step_verbose(counts, item_classes):
 
     return (class_marginals, error_rates)
 
-def e_step(counts, class_marginals, error_rates):
+def e_step(counts_tiled, class_marginals, error_rates):
     """
     Determine the probability of each item belonging to each class,
     given current ML estimates of the parameters from the M-step
     See equation 2.5 in Dawid-Skene (1979)
 
     Inputs:
-      counts: Array of how many times each rating was given
-          by each rater for each item
+      counts_tiled: A matrix of how many times each rating was given
+          by each rater for each item, repeated for each class to make matrix
+          multiplication fasterr. [nItems, nRaters, nClasses, nClasses]
```

Comment:
Can you rename nClasses to distinguish between observation vs truth

Reply:
Sure. What would you suggest I relabel it to? `nClasses` refers to the number of classes, in the binary case it's 2 and in the toxicity case it's 5 etc.

Reply:
I updated the doc string. Hopefully it's a bit clearer now, but happy to add more context.